### PR TITLE
Add Alertmanager configuration

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR mainly adds two things:
- The minimal configuration to run Alertmanager
- The minimal documentation to set-up PipeCD' monitoring system

Both gonna be improved little by little.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/pipecd/values.yaml [hunk 1]
```diff
@@ -187,6 +196,19 @@ prometheus:
...
+        - name: Instances
+          rules:
+            - alert: InstanceDown
+              expr: up == 0
+              for: 5m
+              labels:
+                severity: page
+              annotations:
+                description: '{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes.'
+                summary: 'Instance {{ $labels.instance }} down'
```
Comment (nghialv) [inline manifests/pipecd/values.yaml:211]: Can this be loaded from external files like Grafana dashboards?

Comment (nakabonne) [reply manifests/pipecd/values.yaml:211]: I'll look up

**Comment by nakabonne** [comment]
Found a problem; hold on
/hold

**Comment by nakabonne** [comment]
To honor https://github.com/pipe-cd/pipe/pull/1994#discussion_r637817581, I settled on creating a brand new Configmap for alerting_rules.yml. This way wouldn't be supposed by the Prometheus chart, so I'm running into a reload configuration issue. Hopefully we can merge configmap with the original Configmap defined [here](https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/templates/server/cm.yaml), but I'm stuck with that; put it off.

**Comment by nghialv** [comment]
👀

**Comment by nghialv** [comment]
👍  Let's go with this for now.
/lgtm

**Comment by khanhtc1202** [comment]
📈 
/approve

**Pull request merged**